### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by JKtranslator**
Update FAQ

### CODE REVIEW

File: docs/misc/FAQ.md [hunk 1]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
 
 ## Chain Data
 
-### How much ALPH are in circulation?
+### **How much ALPH are in circulation?[​](https://docs.alephium.org/frequently-asked-questions#how-much-alph-are-in-circulation)**
```
Comment (nop33) [inline docs/misc/FAQ.md]: I don't think adding bold in a header does anything. Also, this is not how you set links. You probably want something like this:

```suggestion
### [How much ALPH are in circulation?](https://docs.alephium.org/frequently-asked-questions#how-much-alph-are-in-circulation)
```

File: docs/misc/FAQ.md [hunk 2]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
 
-You can find the circulating supply on Alephium [Explorer](https://explorer.alephium.org) or by using the [circulating ALPH endpoint](https://backend.mainnet.alephium.org/infos/supply/circulating-alph).
+You can find the circulating supply on Alephium [Explorer](https://explorer.alephium.org/) or by using the [circulating ALPH endpoint](https://backend.mainnet.alephium.org/infos/supply/circulating-alph).
 
-### How is the circulating supply calculated?
+
+### **How is the circulating supply calculated?[​](https://docs.alephium.org/frequently-asked-questions#how-is-the-circulating-supply-calculated)**
```
Comment (nop33) [inline docs/misc/FAQ.md]: Same comment as above

File: docs/misc/FAQ.md [hunk 3]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
-- All locked ALPH from other addresses.
 
-### How can I check wallets balances and which are the top holding addresses?
 
-To check the balance of a specific wallet, the [Explorer](https://explorer.alephium.org) is your go-to.  
-Community members have created [Alph Richlist](https://alph-richlist.vercel.app/) that lists the top holding addresses.
+* The full balance (locked & unlocked) of private sale participants, project and team-controlled addresses. According to CMC: _"tokens are generally only counted as circulating after they leave the original reserve wallet (i.e. outbound transfers are more representative of an intent to bring the coin into circulation rather than a mere unlock)"_.
+* All locked ALPH from other addresses.
+
+
```
Comment (nop33) [inline docs/misc/FAQ.md]: No need for double breakline

File: docs/misc/FAQ.md [hunk 4]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
 
-- The full balance (locked & unlocked) of private sale participants, project and team-controlled addresses. According to CMC: _"tokens are generally only counted as circulating after they leave the original reserve wallet (i.e. outbound transfers are more representative of an intent to bring the coin into circulation rather than a mere unlock)"_.
-- All locked ALPH from other addresses.
 
-### How can I check wallets balances and which are the top holding addresses?
 
-To check the balance of a specific wallet, the [Explorer](https://explorer.alephium.org) is your go-to.  
-Community members have created [Alph Richlist](https://alph-richlist.vercel.app/) that lists the top holding addresses.
+* The full balance (locked & unlocked) of private sale participants, project and team-controlled addresses. According to CMC: _"tokens are generally only counted as circulating after they leave the original reserve wallet (i.e. outbound transfers are more representative of an intent to bring the coin into circulation rather than a mere unlock)"_.
+* All locked ALPH from other addresses.
```
Comment (nop33) [inline docs/misc/FAQ.md]: Why changing the `-` for `*`?

File: docs/misc/FAQ.md [hunk 5]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
+
+
+### **How can I check wallets balances and which are the top holding addresses?[​](https://docs.alephium.org/frequently-asked-questions#how-can-i-check-wallets-balances-and-which-are-the-top-holding-addresses)**
```
Comment (nop33) [inline docs/misc/FAQ.md]: Same comment as above regarding links

File: docs/misc/FAQ.md [hunk 6]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
+
+To check the balance of a specific wallet, the [Explorer](https://explorer.alephium.org/) is your go-to. \
```
Comment (nop33) [inline docs/misc/FAQ.md]: The `\` at the end is probably a typo

File: docs/misc/FAQ.md [hunk 7]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
+Community members have created [Alph Richlist](https://alph-richlist.vercel.app/) that lists the top holding addresses, alongside Alph.Pro, a comprehensive website/tool that offers data on all things DeFi on Alephium, including NFTs, the Richlist, Liquidity pools and more.
 
 In addition, you can find the wrapped ALPH/erc20 holders at [etherscan.io](https://etherscan.io/token/0x590F820444fA3638e022776752c5eEF34E2F89A6#balances)
 
-### What is the minimum Transaction Fee?
+
+### **What is the minimum Transaction Fee?[​](https://docs.alephium.org/frequently-asked-questions#what-is-the-minimum-transaction-fee)**
```
Comment (nop33) [inline docs/misc/FAQ.md]: I won't keep commenting the same thing for each header, but the same applies :)

File: docs/misc/FAQ.md [hunk 8]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
 
 Currently, the minimum transaction fee is set at `0.002` ALPH to prevent network DoS attacks. In the future, this minimum fee can be lowered, with the lowest possible minimum transaction fee being `0.00000000000001` ALPH on Alephium. The exact fee charged depends on the number of inputs (UTXOs) and signers involved in the transaction.
 
-### How many Transactions Per Second (TPS) are possible on Alephium?
 
-Alephium Mainnet can currently support over 400 TPS with 16 shards. It can scale up over 10k TPS by increasing the number of shards as necessary.  
+### **How many Transactions Per Second (TPS) are possible on Alephium?[​](https://docs.alephium.org/frequently-asked-questions#how-many-transactions-per-second-tps-are-possible-on-alephium)**
+
+Alephium Mainnet can currently support over 400 TPS with 16 shards. It can scale up over 10,000+ TPS by increasing the number of shards as necessary. \
```
Comment (nop33) [inline docs/misc/FAQ.md]: Is there a reason for the `\`?

File: docs/misc/FAQ.md [hunk 9]
```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
...
+
+
+### **Is there a DEX on Alephium?[​](https://docs.alephium.org/frequently-asked-questions#is-there-a-dex-on-alephium)**
 
 Alephium has a [DEX prototype](https://alephium.github.io/alephium-dex/#/swap) running on Testnet. Learn more about it in [the DEX prototype article](https://medium.com/@alephium/dex-prototype-live-on-testnet-bac5e7d095ce).
 
 The [DEX contracts](https://github.com/alephium/alephium-dex/tree/master/contracts) have been thoroughly tested by the core developers, making them reliable and efficient. They can easily be forked and used for projects.
 
-### Is there any dApps on Alephium?
+Alephium’s DeFi is continuously expanding and you can check all deployed DEXes on [Coingecko ](https://www.coingecko.com/en/coins/alephium) and on [Alph.land](alph.land).
```
Comment (nop33) [inline docs/misc/FAQ.md]: There's an empty space in `[Coingecko ]`

**Review by nop33 — CHANGES_REQUESTED** [review]

File: docs/misc/FAQ.md (hunk 1 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: removed all bold from the headers and fixed the links

File: docs/misc/FAQ.md (hunk 3 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: removed all

File: docs/misc/FAQ.md (hunk 9 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: fixed. thanks

File: docs/misc/FAQ.md (hunk 8 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: removed all

File: docs/misc/FAQ.md (hunk 6 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: removed all

File: docs/misc/FAQ.md (hunk 4 above)
Comment (JKtranslator) [reply docs/misc/FAQ.md]: removed all

**Comment by JKtranslator** [comment]
Will close this and open a new one.

**Pull request closed without merging**